src/autoComplete/autoCompleteList.tsx

```diff
@@ -139,11 +137,7 @@ extends Component <AutoCompleteListProps, AutoCompleteListState> {
               status={status}
               select={this.onSelect(index)}
               onDoneAnimationEnd={this.props.onDoneAnimationEnd}
-            >
-              <div>
-                {this.props.renderItem({ item, index })}
```

Comment:
So how do we do instead?

Reply:
you should add this in the `breaking change` list then